openedx/adg/admin_dashboard/webinars/api/views.py

```diff
@@ -59,3 +87,27 @@ def delete(self, request, pk):
         webinar_to_delete = get_object_or_404(Webinar, pk=pk)
         webinar_to_delete.delete()
         return Response({}, status=HTTP_200_OK)
+
+
+class WebinarPublishView(APIView):
+    """
+    View to clone a webinar
+    """
+    permission_classes = (IsAuthenticated,)
```

Comment:
I think we should create a new permission as well which will check whether a user is admin or not.